test/api/endpoints/lossClaim/list/v4/request/ListLossClaimsParserSpec.scala

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package api.endpoints.lossClaim.list.v4.request
+
+import api.endpoints.lossClaim.domain.v3.{ TypeOfClaim, TypeOfLoss }
+import api.models.domain.{ Nino, TaxYear }
+import api.models.errors._
+import support.UnitSpec
+
+class ListLossClaimsParserSpec extends UnitSpec {
+
+  private val nino       = "AA123456B"
+  private val businessId = "XAIS01234567890"
+
+  implicit val correlationId: String = "X-123"
+
+  trait Test extends MockListLossClaimsValidator {
+    lazy val parser = new ListLossClaimsParser(mockValidator)
+  }
+
+  "parse" when {
+    "valid input" should {
+      "work where parameters are provided" in new Test {
+        val inputData: ListLossClaimsRawData =
+          ListLossClaimsRawData(
+            nino = nino,
+            taxYearClaimedFor = "2017-18",
```

Comment:
Since we still have MinTaxYearValidation, and minimumTaxYearLossClaim = 2020, this is an invalid input. taxYearClaimedFor should be changed to 2020 or above.